framework/grid/GridView.php

```diff
@@ -503,7 +503,7 @@ public function renderTableRow($model, $key, $index)
         foreach ($this->columns as $column) {
             $cells[] = $column->renderDataCell($model, $key, $index);
         }
-        if ($this->rowOptions instanceof Closure) {
+        if (is_callable($this->rowOptions)) {
```

Comment:
@klimov-paul: is it also risk-prone when using is_callable($this->rowOptions, true) in which syntax_only is set to true (If set to TRUE the function only verifies that name might be a function or method, see http://php.net/manual/en/function.is-callable.php#refsect1-function.is-callable-examples)